WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1243,10 +1241,18 @@ FieldBorder                 = @InputBorder@  @BorderBase@
 AppTabBottomBorder          = border-bottom-width:0px;
 TabBottomBorder             = border-bottom-width:1px; border-bottom-style:solid; @bColor@:black;
 
-WindowOuterBorder           = background-color:@darken(AppC,17)@;   @PopupShadow@           @roundCorners(4px)@         @MediumOutsetBorder@
-LightWindowOuterBorder      = background-color:@darken(AppC,5)@;    @PopupShadow@           @roundCorners(4px)@         @MediumOutsetBorder@
-WindowInnerBorder           = background-color:@lighten(AppC,25)@;  padding:8px 8px 4px;    @roundCorners(0px 0px 4px 4px)@     border-top:1px solid @darken(AppC,40)@;
-LightWindowInnerBorder      = background-color:@lighten(AppC,50)@;  padding:8px 8px 4px;    @roundCorners(0px 0px 4px 4px)@     border-top:1px solid @darken(AppC,40)@;
+WindowOuterBorder           = @PanelBg@   @PopupShadow@   @roundCorners(3px)@; padding: @SkinWrapperPadding@;
+LightWindowOuterBorder      = @WindowOuterBorder@
+WindowInnerBorder           = padding-top: @SkinWrapperPadding@;
+LightWindowInnerBorder      = @WindowInnerBorder@
+WindowButtonBar             = padding-top: @SkinWrapperPadding@;
+
+DialogBodyText              = font-size:1.1rem;
+DialogBody                  = @DialogBodyText@; cursor:default;
+DialogMessageBody           = @DialogBodyText@; width:450px; max-height:400px; overflow:auto;
```

Comment:
Well, I am not really sure of this, I have just taken this rule from current css file and put into property string. Also I think it's fair to have width of dialog as  per UX, for even for less content and big content, we are showing same sized dialog, and size is not based on it's content. 

However we can add `max-width: 100% or 80%` check for mobile, but even that's not needed now, I feel. 

Let me know your suggestions.

- Thanks